src/gluonts/model/lstnet/_network.py

```diff
@@ -0,0 +1,269 @@
+        ar = self.ar_fc(ar_x)
+        ar = F.reshape(ar, shape=(-1, self.num_series))  # NC
+        return ar
+
+    def hybrid_forward(
+        self, F, past_target: Tensor, past_observed_values: Tensor
+    ) -> Tensor:
+        scaled_past_target, _ = self.scaler(
+            past_target.slice_axis(
+                axis=2, begin=-self.context_length, end=None
+            ),
+            past_observed_values.slice_axis(
+                axis=2, begin=-self.context_length, end=None
+            ),
+        )
+        c = self.cnn(scaled_past_target)
+        c = self.dropout(c)
+        c = F.transpose(c, axes=(0, 2, 1))  # NTC
+        r, _ = self.rnn.unroll(
+            inputs=c,
+            length=min(self.conv_out, self.context_length),
+            layout="NTC",
+            merge_outputs=True,
+            begin_state=self.rnn.begin_state(
+                func=F.zeros,
+                dtype=self.dtype,
+                batch_size=c.shape[0] if isinstance(c, mx.nd.NDArray) else 0,
+            ),
+        )
+        r = F.squeeze(
+            F.slice_axis(r, axis=1, begin=-1, end=None), axis=1
+        )  # NC
+        s = self._skip_rnn_layer(F, c)
+        fc = self.fc(F.concat(r, s, dim=1))
+        ar = self._ar_highway(F, past_target)
+        out = fc + ar
+        if self.output_activation is None:
+            return out
+        return (
+            F.sigmoid(out)
+            if self.output_activation == "sigmoid"
+            else F.tanh(out)
+        )
+
+
+class LSTNetTrain(LSTNetBase):
+    @validated()
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+        self.loss_fn = loss.L1Loss()
+
+    def hybrid_forward(
+        self,
+        F,
+        past_target: Tensor,
+        past_observed_values: Tensor,
+        future_target: Tensor,
+    ) -> Tensor:
+        """
+        Computes the training loss for LSTNet for multivariate time-series.
+        All input tensors have NCT layout.
+
+        Parameters
+        ----------
+        F
+        past_target: (batch_size, num_series, context_length)
+        past_observed_values: (batch_size, num_series, context_length)
+        future_target: (batch_size, num_series, prediction_length)
+
+        Returns
+        -------
+        Tensor
+            Loss value of shape (1,)
+        """
+
+        ret = super().hybrid_forward(F, past_target, past_observed_values)
+        # get the last time horizon
+        future_target = F.slice_axis(future_target, axis=2, begin=-1, end=None)
+        loss = F.mean(self.loss_fn(ret, future_target), axis=-1)
+        return loss
+
+
+class LSTNetPredict(LSTNetBase):
+    def hybrid_forward(
+        self, F, past_target: Tensor, past_observed_values: Tensor
+    ) -> Tensor:
+        """
+        Returns the LSTNet predictions. All tensors have NCT layout.
+
+        Parameters
+        ----------
+        F
+        past_target: (batch_size, num_series, context_length)
```

Comment:
The description of the arguments should be in an indented new line, similarly to how it is in the estimator

Reply:
Think I looked at [this](https://github.com/awslabs/gluon-ts/blob/master/src/gluonts/model/deepar/_network.py#L363). Will indent.

Reply:
Yeah, that one should be fixed too...